src/visualization/plot_recipes.jl

```diff
@@ -290,3 +293,21 @@ end
                     slice_axis_intercept=slice_axis_intercept)
 end
 
+
+# Create a PlotData2D plot directly from a TimeIntegratorSolution for convenience
+#
+# Note: This is an experimental feature and may be changed in future releases without notice.
+#
+# Note: If you change the defaults values here, you need to also change them in the PlotData2D
+#       constructor.
+@recipe function f(sol::TimeIntegratorSolution;
+                   solution_variables=cons2prim,
+                   grid_lines=true, max_supported_level=11, nvisnodes=nothing, slice_axis=:z,
+                   slice_axis_intercept=0)
+  return PlotData2D(sol;
+                    solution_variables=solution_variables,
+                    grid_lines=grid_lines, max_supported_level=max_supported_level,
+                    nvisnodes=nvisnodes, slice_axis=slice_axis,
+                    slice_axis_intercept=slice_axis_intercept)
+end
```

Comment:
This will also become obsolete if the `union` above works

Reply:
Removed in 17d22ba7.